commander/src/bootstrapping/commands/genesis-block/create.ts

```diff
@@ -84,6 +114,9 @@ export abstract class BaseGenesisBlockCommand extends Command {
 				schema: a.schema,
 			})),
 			chainID: Buffer.from(app.config.genesis.chainID, 'hex'),
+			height,
+			timestamp,
+			previousBlockID,
```

Comment:
very minor thing but you could do
previousBlockIDString ? Buffer.from(previousBlockIDString, 'hex') : undefined